board/can.h

```diff
@@ -21,7 +22,7 @@ void can_init(CAN_TypeDef *CAN, int silent) {
     CAN->BTR |= CAN_BTR_SILM | CAN_BTR_LBKM;
   #endif
 
-  if (silent) {
+  if (!controls_allowed) {
```

Comment:
Good to know. 
I was convinced they were having the same behavior because silent defaulted to on with PANDA_SAFETY, but as soon as the usb  'controls enabled' message was received (to enable or disable), silent would be disabled and never reenabled.
https://github.com/commaai/panda/pull/23/files#diff-6cd306808dae15d8f629c38dc9e817a0L655